### DISCUSSION TITLE
Need to Read Google-Service.json from server

### DISCUSSION THREAD

**Discussion by pavansha** [invertase/react-native-firebase#6243] [Q&A]
How can I add a google service file dynamically in the project

**Comment by mikehardy** [comment]
You cannot, it is not supported by the native SDKs.

**Reply by pavansha** [reply]
it is possible in native android?

**Reply by mikehardy** [reply]
>  it is not supported by the native SDKs.

No, sorry

**Reply by pavansha** [reply]
ok
